### DISCUSSION TITLE
A little bit suggestion

### DISCUSSION THREAD

**Discussion by bensonxu770729** [jellyfin/jellyfin#8912] [General] [answered]
I am use the Jellyfin 10.8.6, everything is fine, but only have one suggestion:

As a film collector, when I click one actor's name/photo and go into the actor's page, the film list is only could sort by alphabet, does it better sort by the film's release year? or give us a option to choose

Thanks

**Comment by nielsvanvelzen** [comment, marked as answer]
Feature requests should be posted to https://features.jellyfin.org

**Comment by Nordlicht-13** [comment]
Vote here: https://features.jellyfin.org/posts/2636/full-filmography-of-actors
